docs/scripts/util/getCatalog.ts

```diff
@@ -48,6 +49,30 @@ function getCatalog() {
   const catalog: Catalog = {} as Catalog;
   for (const [componentName, [node]] of source.getExportedDeclarations()) {
     let properties: Properties = {};
+    const children = componentsWithChildren[componentName];
```

Comment:
This code adds nested components into the catalog. New primitives need to be added to the catalog.d.ts and componentsWithChildren